kod/object/active/holder/room.kod

```diff
@@ -886,7 +889,7 @@ messages:
          return RID_KOCATAN;
       }
 
-      if (piRoom_num = RID_GUILDH12) or (piRoom_num = RID_GUILDH13)
+      if (piRoom_num = RID_GUILDH12) or (piRoom_num = RID_GUILDH13) or (lRoom_data = RID_KOC_INN)
```

Comment:
You don't need lRoom_data until here, so don't call GetLocation until this point.  Also, can you think of a more meaningful name?

Also, please add a comment explaining why this check is here.